docs/workflow/analyze.md

```diff
@@ -1,20 +1,140 @@
+
+*(new) particle-based surface reconstruction*
+<p><video src="https://sci.utah.edu/~shapeworks/doc-resources/mp4s/particles_reconstruction.mp4" autoplay muted loop controls style="width:100%"></p>
+
+
+You can export any mesh using `File -> Export Current Mesh ...`. 
+
+
+## Statistical Analysis 
+
+Statistical analysis is performed using principal component analysis (PCA), where the mean and modes of shape variation are computed based on the optimized correspondence model. Automated animation is allowed to watch the shape morph at various standard deviations along a user-selected variation mode. 
+
+*Animating the shape variation along with a principal component while passing through the mean shape*
+
+<p><video src="https://sci.utah.edu/~shapeworks/doc-resources/mp4s/femur_first_mode.mp4" autoplay muted loop controls style="width:100%"></p>
+
+
+You can also export the eigenvalues and each shape's PCA loadings in an excel sheet for further analysis. 
+
+## Dynamic Loading
+
+ShapeWorksStudio support load-on-demand, which makes it more scalable by supporting loading in order of 100s of samples. For instance, an old small project that took 10+ seconds to load now starts instantly in Studio.
+
+
+*(old) without dynamic loading*
+
+<p><video src="https://sci.utah.edu/~shapeworks/doc-resources/mp4s/studio_load_old.mp4" autoplay muted loop controls style="width:100%"></p>
+
+
+*(new) with dynamic loading*
+
+<p><video src="https://sci.utah.edu/~shapeworks/doc-resources/mp4s/studio_load_new.mp4" autoplay muted loop controls style="width:100%"></p>
+
+
+## Group Differences
+
+
+If there are groups in the data, ShapeWorksStudio can visualize significant group differences. 
+
+
+*Statistical group differences for characterizing scapular morphology in Hill-Sachs patients*
+![Template-deformation approach for surface reconstruction](../img/workflow/scapula_mean_gdiff.png)
+
+
+If you are using an XML file to [load your data into ShapeWorksStudio](#running-shapeworksstudio), a group id (1 or 2) should be given to each sample. If you are using a spreadsheet, specify group columns in the `data` (first) sheet in the spreadsheet with the prefix `group_`, which enables adding multiple groups to your project and selecting the group-of-interest within ShapeWorksStudio. 
+
+*Groups as columns in the project spreadsheet*
+![Template-deformation approach for surface reconstruction](../img/workflow/sheet_groups.png)
+
+
+*In the Group tab of the Analyze panel, choose which column to indicate the group set of interest*
+![Template-deformation approach for surface reconstruction](../img/workflow/studio_select_group_set.png)
+
+
+*Then, select which groups you would like to compare (now support more than two groups)*
+![Template-deformation approach for surface reconstruction](../img/workflow/studio_select_group.png)
+
+## Running ShapeWorksStudio
```

Comment:
"Running ShapeWorksStudio" should be somewhere higher up

Reply:
Done